app/scripts/controllers/landingPage.js

```diff
@@ -51,10 +52,21 @@ angular.module('openshiftConsole')
 
     AuthService.withUser().then(function() {
       var includeTemplates = !_.get(Constants, 'ENABLE_TECH_PREVIEW_FEATURE.template_service_broker');
-      Catalog.getCatalogItems(includeTemplates).then(function(items) {
+      Catalog.getCatalogItems(includeTemplates).then(_.spread(function(items, errorMessage) {
+        if (errorMessage) {
+          var alert = {
+            name: 'error-loading-catalog-items',
+            data: {
+              type: 'warning',
```

Comment:
Suggest an error here rather than a warning.

I'm surprised DataService doesn't show an error if the list fails. Won't there be two errors?